telemetry/specs/INT.mdk

```diff
@@ -998,7 +997,7 @@ INT Metadata Header and Metadata Stack:
     0                   1                   2                   3
     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   |  Ver  |Rep|C|E|  R  | InsCnt=1|  MaxHopCnt=16 | TotalHopCnt=3 |
+   |  Ver  |Rep|C|E|  R  | InsCnt=1|   Reserved    |RemainingHopC=5|
```

Comment:
Could you please also add M bit here? I missed this example in my commit.

Reply:
Done